VAMobile/src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -87,6 +91,14 @@ export const HomeScreen: FC<HomeScreenProps> = ({ navigation }) => {
     }, [dispatch, smInDowntime, userAuthorizedServices?.secureMessaging]),
   )
 
+  useFocusEffect(
+    useCallback(() => {
+      if (userAuthorizedServices?.decisionLetters && !lettersInDowntime) {
```

Comment:
I don't think this authorization is required for this info?

Reply:
Hmm initially I though it would either be `decisionLetters` or `lettersAndDocuments` required for it. I noticed the only other place we're calling it is [here](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/4022d1a21156c113128e92ab90bfbcaa385217a9/VAMobile/src/screens/BenefitsScreen/Letters/BenefitSummaryServiceVerification/BenefitSummaryServiceVerification.tsx#L57), and we're not even checking any authorized service 🤔. I'll do some digging and see